### DISCUSSION TITLE
Не работает с камерой-лампочкой

### DISCUSSION THREAD

**Discussion by hemml** [Arkady23/V380-Web#65] [General]
Приветствую!

Не могу получить telnet (и вообще ничего не могу получить) от камеры, которая вкручивается в патрон как лампочка. На коробке написано V380. 
В приложении показывает версии AppEV2W_VA3_V1.3.0.0_20230519 и Hw_AWT3710D_XHR_V1.0_WF_20230519
Пробовал через ceshi.ini - камера говорит по-китайски, но ничего не меняется.
Открытые порты TCP: 8800 и 9800

**Comment by hristo-atanasov** [comment]
Exactly the same with my camera, with the exception, that my camera is not a light bult.

![image](https://github.com/Arkady23/V380-Web/assets/20243170/8125e188-1a0b-4eb0-b65f-47e0ed7c4a87)
![image](https://github.com/Arkady23/V380-Web/assets/20243170/00c404ac-09ad-4f23-8360-a32e8a7e652f)
![image](https://github.com/Arkady23/V380-Web/assets/20243170/1810dba2-e8b4-4475-aec9-253a1bd2cdd3)

**Reply by Arkady23** [reply]
Наверное надо пробовать новую прошивку менять на старую.
Probably it is necessary to try to change the new firmware to the old one.

**Reply by hristo-atanasov** [reply]
Can you give us a hint how we can upload old firmware and maybe where we can find it?

**Reply by Arkady23** [reply]
There are articles on the Internet, as someone has already done. To be honest, I have not encountered this. It uses a flash memory programmer. And someone should share it with the old firmware.

**Reply by hristo-atanasov** [reply]
Cool .. I have a flash memory programmer, so now I just need I fo about the software that I have to use and link to the firmware.

**Reply by Arkady23** [reply]
Don't you have a telnet?

**Reply by hristo-atanasov** [reply]
Telnet to the camera?

**Reply by Arkady23** [reply]
Yes

**Reply by hristo-atanasov** [reply]
I haven't tried. What if I have?

**Reply by Arkady23** [reply]
The web-server is installed if there is a telnet-server on the camera. If there is no access to the camera via telnet, then yes.... you can try switching to the old firmware with telnet, if possible.

**Reply by hristo-atanasov** [reply]
Unfortunately I don't have .. At least on ports 23, 22 and 21.

**Reply by acong1209** [reply]
mr hristo apakah cctv camera di atas sudah done.bagaimana cara nya saya juga sama punya kasus camera seperti di atas rusak.ic epromnya harus di ganti,sedangakn saya tidak punya firmware full dumpnya tidak ada,mohon bantuanya.terima kasih.

**Reply by Arkady23** [reply]
Try applying this patch. If it works, it will download important files from the camera to the SD card.

[EV2W_VA3-backup.zip](https://github.com/user-attachments/files/25727389/EV2W_VA3-backup.zip)

**Comment by welshi32** [comment]
Тоже самое. Телнет не открывается. Помогите пожалуйста

**Comment by Arkady23** [comment]
Появился новый способ установки веб на камеру. В ближайшее время опишу на главной странице.
